### ISSUE TITLE
API inconsistent about how to write ID or Id

### ISSUE THREAD

**Issue by ddresch** [rebeccahughes/react-native-device-info#314]
I ran into the problem the method `getUniqueId()` was not found. Checking the README you can see the example for this method is incorrect.

```JS
const uniqueId = DeviceInfo.getUniqueId();

// iOS: "FCDBD8EF-62FC-4ECB-B2F5-92C9E79AC7F9"
// Android: "dd96dec43fb81c97"
// Windows: ?
```

The method is called `getUniqueID()` so changing the Docs is the simple way but I think it's better to make it more consistent across the API.

```JS
getBundleId()
getDeviceId()
getInstanceID()
getUniqueID()
```

What do you think?

**Comment by machour** [comment]
Hi @ddresch and thank you for spotting this!

For now, and in order to avoid BC breaks, only the documentation should be updated. I'd love if you can open a new PR to fix this.

I've started a checklist/discussion for the 1.0.0 (where we can break whatever we want), definitely adding your renaming proposal there! #307

**Comment by ddresch** [comment]
PR is done ;-) #315

**Comment by machour** [comment]
And it's merged, thank you very much!

**Comment by ddresch** [comment]
Perfect, closed this issue as it's referenced in #307
